modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx

```diff
@@ -193,43 +201,38 @@ export default function ActionBuilder({
 
 		const {items} = (await response.json()) as {items: ObjectField[]};
```

Comment:
Since the `json()` is `any` you can just set the constant type:
```suggestion
		const {items}: {items: ObjectField[]} = await response.json();
```

Reply:
I don't need to do this change anymore because you've created the `getObjectFields()` function and is using it here :relaxed: